status/flowcell.py

```diff
@@ -108,11 +108,36 @@ def get(self, flowcell_id):
             entry.value['plist'] = self._get_project_list(entry.value)
             # list of project_names -> to create links to project page and bioinfo tab
             project_names = {project_name: self._get_project_id_by_name(project_name) for project_name in entry.value['plist']}
+            # Prepare a summary table for total project yields in each lane
+            fc_project_yields = dict()
+            for lane_nr in sorted(entry.value.get('lanedata').keys()):
+                fc_project_yields_lane_list = []
+                lane_details = entry.value['lane'][lane_nr]
+                total_lane_yield = int(entry.value['lanedata'][lane_nr]['clustersnb'].replace(',',''))
+                unique_projects = list(set(i['Project'] for i in lane_details))
+                threshold = thresholds.get(entry.value.get('run_mode', ''), 0)
+                for proj in unique_projects:
+                    if proj == 'default':
+                        modified_proj_name = 'undetermined'
+                    else:
+                        modified_proj_name = proj.replace('__','.')
+                    sum_project_lane_yield = sum(int(i['clustersnb'].replace(',','')) for i in lane_details if i['Project']==proj)
```

Comment:
`item` would also work here but `lane` might be more informative?
```suggestion
                    sum_project_lane_yield = sum(int(lane['clustersnb'].replace(',','')) for lane in lane_details if lane['Project']==proj)
```